src/fontstash.h

```diff
@@ -282,18 +294,40 @@ int fons__tt_getGlyphIndex(FONSttFontImpl *font, int codepoint)
 }
 
 int fons__tt_buildGlyphBitmap(FONSttFontImpl *font, int glyph, float size, float scale,
-							  int *advance, int *lsb, int *x0, int *y0, int *x1, int *y1)
+							  int oversampleH, int oversampleV,
+							  int *advance, int *lsb, int *x0, int *y0, int *x1, int *y1,
+							  float *subX, float *subY)
 {
 	FONS_NOTUSED(size);
 	stbtt_GetGlyphHMetrics(&font->font, glyph, advance, lsb);
-	stbtt_GetGlyphBitmapBox(&font->font, glyph, scale, scale, x0, y0, x1, y1);
+	stbtt_GetGlyphBitmapBox(&font->font, glyph, scale * oversampleH, scale * oversampleV, x0, y0, x1, y1);
+	*subX = stbtt__oversample_shift(oversampleH);
+	*subY = stbtt__oversample_shift(oversampleV);
 	return 1;
 }
 
 void fons__tt_renderGlyphBitmap(FONSttFontImpl *font, unsigned char *output, int outWidth, int outHeight, int outStride,
-								float scaleX, float scaleY, int glyph)
+								float scaleX, float scaleY, int oversampleH, int oversampleV, int glyph)
 {
-	stbtt_MakeGlyphBitmap(&font->font, output, outWidth, outHeight, outStride, scaleX, scaleY, glyph);
+	stbtt_MakeGlyphBitmapSubpixel(&font->font,
```

Comment:
Put more params on one line to follow the coding style, i.e. break after outStride.